content/docs/04-clusters/01-public-cloud/03-azure.md

```diff
@@ -41,136 +41,24 @@ The following prerequisites must be met before deploying a workload cluster in A
 
 3. You should have an Infrastructure cluster profile created in Palette for Azure cloud.
 
-# Creating an Azure Cloud Account
+# Section Highlights
+
+This section of the document describes the following capabilities:

```

Comment:
```suggestion
To learn more about Palette and Azure integrations check out the following resources:
```
This helps the reader understand that the resources below are resources for Azure and Palette.